charts/aws-vpc-cni/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: aws-vpc-cni
-version: 1.13.2
-appVersion: "v1.13.2"
+version: 1.13.3
+appVersion: "v1.13.3"
```

Comment:
Why is the appVersion updated? We generally update this during release..

Reply:
ack